packages/reports/src/sass/app/modules/reports.scss

```diff
@@ -717,11 +717,36 @@ $side-padding: 1.2em;
         }
     }
 
-    &__contract-types-selector {
-        padding: 0 1.6rem;
-    }
-    &__contract-types-selector__container {
+    &__contract-types-selector-container {
         max-width: 22.4rem;
+
+        &--mobile {
+            width: 100%;
+            padding: 0 1.6rem;
+        }
+    }
+    &__accumulator-container {
+        display: flex;
+        flex-direction: row;
```

Comment:
```suggestion

```
@henry-deriv `flex-direction: row` is the default one for flex, no need for this one )